drivers/i2c/i2c_ll_stm32_v2.c

```diff
@@ -224,9 +224,17 @@ int i2c_stm32_target_register(const struct device *dev,
 
 	if (!data->slave_cfg) {
 		data->slave_cfg = config;
+	if( data->slave_cfg->flags == I2C_SLAVE_FLAGS_ADDR_10_BITS )
+	{
+		LL_I2C_SetOwnAddress1(i2c, config->address, LL_I2C_OWNADDRESS1_10BIT);
+		LOG_DBG("i2c: slave #1 registered with 10-bit address");
+	}
+	else
+	{
+		LL_I2C_SetOwnAddress1(i2c, config->address << 1U, LL_I2C_OWNADDRESS1_7BIT);
+		LOG_DBG("i2c: slave #1 registered with 7-bit address");
+	}
```

Comment:
Please fix indentation:
```suggestion
		if (data->slave_cfg->flags == I2C_SLAVE_FLAGS_ADDR_10_BITS)	{
			LL_I2C_SetOwnAddress1(i2c, config->address, LL_I2C_OWNADDRESS1_10BIT);
			LOG_DBG("i2c: slave #1 registered with 10-bit address");
		} else {
			LL_I2C_SetOwnAddress1(i2c, config->address << 1U, LL_I2C_OWNADDRESS1_7BIT);
			LOG_DBG("i2c: slave #1 registered with 7-bit address");
		}
```